app/components/CarouselCard.tsx

```diff
@@ -119,11 +127,21 @@ const $meta: ViewStyle = {
   marginBottom: spacing.medium,
 }
 
+const $mr: TextStyle = {
+  marginRight: spacing.large,
```

Comment:
It's still a super new feature, but since we're on RN 0.71 now, could we try using the `gap` property? https://reactnative.dev/docs/flexbox#row-gap-column-gap-and-gap. I think that could simplify this a lot and you could possibly just use `leftButton` for the static version instead of adding the `button` prop.

Reply:
Ooh, gotcha, I didn't notice they were functionally different!